### PR TITLE
feat: card-image-input

### PR DESCRIPTION
**Pull request by 05him**
added card, image, and input components

live URL:  https://05him.github.io/Dark_UI/

### CODE REVIEW

File: assets/empty-heart-badge.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="23" height="19" viewBox="0 0 23 19" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (arjun-computer-geek) [inline assets/empty-heart-badge.svg:1]: is it good to use height and width attribute?

File: components/button/button.html [hunk 2]
```diff
@@ -74,7 +74,51 @@ <h2 class="head">Outline Button</h2>
...
+                <i class="fas fa-sign-out-alt"></i>
+                Logout
+            </button>
+          </div>
+          <iframe
+          class="code-box"
+          src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=27px&ph=21px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%253Cbutton%2520type%253D%2522button%2522%2520class%253D%2522btn%2520btn-primary%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-sign-in-alt%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520Login%250A%2520%2520%2520%2520%2520%2520%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%2520type%253D%2522button%2522%2520class%253D%2522btn%2520btn-primary-outline%2522%253E%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-sign-out-alt%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520Logout%250A%2520%2520%2520%2520%2520%2520%253C%252Fbutton%253E"
```
Comment (arjun-computer-geek) [inline components/button/button.html:103]: you can login to carbon it will give you short link

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+.card-box-vertical:hover,
+.card-box-horizontal:hover {
+  box-shadow: 0.3px 0.3px 3px 0.3px var(--secondary-color);
+}
+
+.card-img {
+  background: var(--demo-bg);
+  border-radius: 15px 15px 0 0;
+  position: relative;
+  padding: 0.4rem;
```
Comment (arjun-computer-geek) [inline components/card/card.css:24]: em is better than rem for margin and padding.

File: components/card/card.html [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Card | DARK-UI</title>
+    <link rel="stylesheet" href="../../doc.css" />
+    <link rel="icon" href="../../assets/favicon.ico">
+    <script src="https://kit.fontawesome.com/d9dd91a45f.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+      <header class=" header">
+      <a href="../../index.html"><span class="header-logo">&#128128DARK-UI</span></a>
```
Comment (arjun-computer-geek) [inline components/card/card.html:14]: you can simply use "/" instead "../../index.html"

**Review by arjun-computer-geek — APPROVED** [review]
other  else looks good to me

File: components/card/card.css [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+}
+
+.card-badge-top-right {
+  position: absolute;
+  left: 83%;
+  top: 5%;
+  background: var(--text-color);
+  border: 10px solid var(--text-color);
+  border-radius: 50px;
```
Comment (aayushjain-code) [inline components/card/card.css:33]: try to use rem values avoid using px

**Review by aayushjain-code — APPROVED** [review]

File: assets/empty-heart-badge.svg (hunk 1 above)
Comment (05him) [reply assets/empty-heart-badge.svg:1]: soory but i dont know about this thing i've just copied the SVG from figma and pasted in one file

File: components/button/button.html (hunk 2 above)
Comment (05him) [reply components/button/button.html:103]: ohh thanks for the advice .. didnt knew it

File: components/card/card.css (hunk 3 above)
Comment (05him) [reply components/card/card.css:24]: thank you for this info.. will now keep it in mind

File: components/card/card.css (hunk 5 above)
Comment (05him) [reply components/card/card.css:33]: ive used rems almost everywhere just for the border thing using px

File: components/card/card.html (hunk 4 above)
Comment (05him) [reply components/card/card.html:14]: thanks a lot for the suggestion, but i like using it that way only

File: components/card/card.css [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+}
+
+.card-box-horizontal .card-btn-box button {
+  width: 95%;
+}
+
+.card-overlay {
+  background: var(--card-overlay-bg);
+  position: absolute;
+  top: -19%;
```
Comment (MuskanChhatrasal) [inline components/card/card.css:123]: Try to use relative units. Else everything looks good to me!!

**Review by MuskanChhatrasal — APPROVED** [review]

File: components/card/card.css (hunk 6 above)
Comment (05him) [reply components/card/card.css:123]: thank you for the advice, btw its relative to the parent component only ..  
what I was wanting to do here was easy with % so I used % here

**Pull request merged**